component/src/main/java/com/synopsys/integration/alert/component/certificates/web/CertificateActions.java

```diff
@@ -69,25 +69,23 @@ public CertificateActions(CertificatesDescriptorKey descriptorKey, Authorization
     @Override
     protected Optional<CertificateModel> findExisting(Long id) {
         return certificateAccessor.getCertificate(id)
-                   .map(this::convertFromDatabaseModel);
+                   .map(this::convertDatabaseModel);
     }
 
     @Override
-    public ActionResponse<MultiCertificateModel> readAllWithoutChecks() {
-        List<CertificateModel> certificates = certificateAccessor.getCertificates().stream()
-                                                  .map(this::convertFromDatabaseModel)
-                                                  .collect(Collectors.toList());
-        return new ActionResponse<>(HttpStatus.OK, new MultiCertificateModel(certificates));
+    protected List<CustomCertificateModel> getDatabaseModels() {
+        return certificateAccessor.getCertificates();
     }
 
     @Override
-    protected ActionResponse<CertificateModel> readWithoutChecks(Long id) {
-        Optional<CertificateModel> model = findExisting(id);
-        if (model.isPresent()) {
-            return new ActionResponse<>(HttpStatus.OK, model.get());
-        }
+    protected MultiCertificateModel createMultiResponseModel(final List<CertificateModel> certificates) {
+        return new MultiCertificateModel(certificates);
+    }
 
-        return new ActionResponse<>(HttpStatus.NOT_FOUND, String.format("Certificate with id:%d not found.", id));
+    @Override
+    protected CertificateModel convertDatabaseModel(CustomCertificateModel databaseCertificateModel) {
```

Comment:
Again, I feel like this method name only chooses to tell one half of the story. Why not `convertToRestModel`? It's also true, but incomplete.

Reply:
It might have been easier to critique the method name in the base class so you only had to do it once.

Reply:
My mistake.